src/developer/web-api/tracker.md

```diff
@@ -611,32 +632,191 @@ Examples for the **FLAT** and the **NESTED** versions of the payload are listed
 }
 ```
 
-### SYNC and ASYNC
+### Create
+
+Make a `POST` to `/api/tracker` with the `importStrategy` set to `CREATE` or `CREATE_AND_UPDATE` and
+a payload as described [here](#payload). 
+
+### Update
 
-For the user, the main difference between importing synchronously rather than asynchronously is the
-immediate response from the API. For the synchronous import, the response will be returned as soon
-as the import finishes with the importSummary. However, for asynchronous imports, the response will
-be immediate and contain a reference where the client can poll for updates to the import.
+Make a `POST` to `/api/tracker` with the `importStrategy` set to `UPDATE` or `CREATE_AND_UPDATE` and
+a payload as described [here](#payload).
 
-For significant imports, it might be beneficial for the client to use the asynchronous import to
-avoid waiting too long for a response.
+The payload must include all fields of the object you are updating, even if they have not been
+modified. The only exception is collections. Items in a collection that should not be changed can be
+omitted, as demonstrated in [update attribute values](#update-data-values) and [update data
+values](#update-data-values).
 
-Examples of the **ASYNC** response is shown below. For **SYNC** response, look at the [importSummary
-section](#webapi_nti_import_summary).
+Updating is not permitted in the following cases:
+* when the tracker object has already been deleted.
+* when dealing with a relationship.
```

Comment:
I am wondering if this is clear enough or it sounds like for example if you are updating an enrollment with a relationship then it is not possible.